CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
-## 1.8.0 (Unreleased)
+## 1.8.0 (March 19, 2020)
```

Comment:
FYI hashi's release automation takes care of setting this to released and appending release date when the release occurs.  We don't usually edit that stuff manually IIRC.